src/test/java/nextstep/subway/acceptance/SectionAcceptanceTest.java

```diff
@@ -38,12 +41,55 @@ void initStations() {
 
     /**
      * Given 노선을 생성한다
-     * When 구간을 생성한다
+     * When 구간을 생성한다. - 새로운 역이 역 사이일 때
      * Then 생성한 노선의 하행선이 변경된다.
      */
-    @DisplayName("지하철 구간을 생성한다.")
+    @DisplayName("지하철 구간을 생성한다. - 새로운 역이 역 사이일 때")
     @Test
     void createSection() {
+        //given
+        지하철_노선도_등록("7호선", "bg-1234", 1L, 3L, 5);
```

Comment:
역 ID를 숫자로 작성하기 보다 역명을 그대로 쓴다면 테스트 가독성을 높여보면 어떨까요?
```suggestion
        지하철_노선도_등록("7호선", "bg-1234", 첫번째역, 세번째역, 5);
```